tsl/test/sql/continuous_aggs.sql

```diff
@@ -1226,3 +1226,72 @@ FROM issue3248 GROUP BY 1,2;
 SELECT
   FROM issue3248 AS m,
        LATERAL(SELECT m FROM issue3248_cagg WHERE avg IS NULL LIMIT 1) AS lat;
+
+-- test that option create_group_indexes is taken into account
+SET client_min_messages = ERROR;
+
+CREATE TABLE test_group_idx (
+time timestamptz,
+symbol int,
+value numeric
+);
+
+select create_hypertable('test_group_idx', 'time');
+
+insert into test_group_idx
+(select generate_series(
+	'2020-01-01',
+    '2020-02-25', INTERVAL '30 sec'
+  ),
+  round(random()*10),
+  random()*5);
+
+create materialized view cagg_index_true
+with (timescaledb.continuous, timescaledb.create_group_indexes = true) as 
+select
+	time_bucket('1 day', "time") as bucket,
+	sum(value),
+	symbol 
+from test_group_idx 
+group by bucket, symbol;
+
+create materialized view cagg_index_false
+with (timescaledb.continuous, timescaledb.create_group_indexes = false) as 
+select
+	time_bucket('1 day', "time") as bucket,
+	sum(value),
+	symbol 
+from test_group_idx 
+group by bucket, symbol;
+
+create materialized view cagg_index_default
+with (timescaledb.continuous) as 
+select
+	time_bucket('1 day', "time") as bucket,
+	sum(value),
+	symbol 
+from test_group_idx 
+group by bucket, symbol;
+
+-- make sure a group index has been created when explicitly asked for
+SELECT format('%I.%I', materialization_hypertable_schema,
+materialization_hypertable_name) AS mat_hyper_cagg_index_true
+FROM timescaledb_information.continuous_aggregates where view_name like 'cagg_index_true';
+
+\gset
+
+\d+ :mat_hyper_cagg_index_true
+
+SELECT format('%I.%I', materialization_hypertable_schema,
+materialization_hypertable_name) AS mat_hyper_cagg_index_false
+FROM timescaledb_information.continuous_aggregates where view_name like 'cagg_index_false';
+
+\gset
```

Comment:
```suggestion
SELECT format('%I.%I', materialization_hypertable_schema,
materialization_hypertable_name) AS mat_hyper_cagg_index_false
FROM timescaledb_information.continuous_aggregates where view_name like 'cagg_index_false' \gset
```
To speed up the queries, instead of executing them twice.